packages/contracts/src/utils/asset_transfer_proxy_utils.ts

```diff
@@ -0,0 +1,63 @@
+import { BigNumber } from '@0xproject/utils';
+import * as Web3 from 'web3';
+import { AssetProxyId } from './types'
+var ethersUtils = require('ethers-utils');
+
+export function zeroPad(value: string, width: number): string {
```

Comment:
`ethereumjs-util` has some helper methods for padding already (for Buffers). I also find it more intuitive/readable to use Buffers for these types of things in general, since that's closer to what's happening at the EVM level.

Reply:
There's also [`String.prototype.padEnd`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padEnd). (After the [leftPad disaster](https://www.theregister.co.uk/2016/03/23/npm_left_pad_chaos/) the JS community woke up and realized that this stuff is needs to become part of the language standard :) )